straxen/plugins/peaklets/peaklets.py

```diff
@@ -302,15 +299,10 @@ def compute(self, records, start, end):
         # Saturation correction using non-saturated channels
         # similar method used in pax
         # see https://github.com/XENON1T/pax/pull/712
-        # Cases when records is not writeable for unclear reason
-        # only see this when loading 1T test data
-        # more details on https://numpy.org/doc/stable/reference/generated/numpy.ndarray.flags.html
-        if not r["data"].flags.writeable:
```

Comment:
If `r["data"].flags.writeable == False`, you will see error like `ValueError: assignment destination is read-only`. So I would not worry about this PR changes the result. Let's see how the reprocessing goes.